config/v1/types_cluster_version.go

```diff
@@ -224,6 +229,17 @@ type UpdateHistory struct {
 // ClusterID is string RFC4122 uuid.
 type ClusterID string
 
+// ClusterVersionArchitecture enumerates valid cluster architectures.
+// +kubebuilder:validation:Enum=multi
+type ClusterVersionArchitecture string
+
+const (
+	// ClusterVersionArchitectureMulti identifies a heterogeneous architecture.
+	// A heterogeneous cluster is a cluster with nodes running on different
```

Comment:
Maybe:

> A heterogeneous cluster is capable of running nodes with different architectures.

or some such to distinguish between "is capable of running nodes with different architectures" (all clusters with heterogeneous-arch release images) and "is actually running nodes with different architectures" (most clusters with heterogeneous-arch release images)?